src/components/Header/TopNav/ServiceLink.js

```diff
@@ -32,7 +34,7 @@ const ServiceLink = ({ path = '/' }) => {
       {isServicePage || isSpecialityPage ? (
         <StyledServiceLink
           to={`/${service}`}
-          hoverColor={isServicePage ? 'black' : 'white'}
+          color={theme.colors[isServicePage ? 'black' : 'white']}
```

Comment:
How about passing in an `isServicePage` prop & then putting this ternary in the CSS? That way, you wouldn't need to import the theme into this component, as we have access via styled components, due to the ThemeProvider that wraps every page.

Reply:
Nice!! Thanks for that. Updated! 👍 and removes the need to import theme 😄